workers/bubbleSort/run.py

```
@@ -0,0 +1,39 @@
+
+import websockets
+import asyncio
+import json
+from bubbleSort import bubbleSort
```

Ruff (0.8.2)

`bubbleSort.bubbleSort` imported but unused

Remove unused import: `bubbleSort.bubbleSort`

(F401)